tests/docker-stacks-foundation/test_packages.py

```diff
@@ -76,10 +76,9 @@
     "hdf5",
     "jupyterhub-singleuser",
     "jupyterlab-git",
-    "mamba[version='<2.0.0']",
-    "notebook[version='>",
+    "mamba",
+    "notebook>",
     "openssl",
-    "pandas[version='>",
```

Comment:
I don't think we need this in both v1 and v2, will remove it as a separate commit today, if it's true.